python/sedona/core/serde/shape/enums.py

```diff
@@ -0,0 +1,22 @@
+from enum import Enum
+
+
+class GeomEnum(Enum):
```

Comment:
@Imbruced Any reason why this only enumerates a few types instead of all types?

Reply:
I used Shape types fro ShapeType enum within package org.apache.sedona.core.formatMapper.shapefileParser.parseUtils.shp or you meant 
```java
        SHAPE(0),
        CIRCLE(1),
        GEOMETRYCOLLECTION(2),
        ENVELOPE(3);
```